_database_api/status.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Status
+layout: default
+---
+
+# Logs
+
+--- row ---
+
+## Get database instances status
+
+--- row ---
+
+`GET https://db-api.scalingo.com/dapi/databases/[:database_id]/instances_status`
```

Comment:
```suggestion
`GET https://$DB_API_URL/api/databases/[:database_id]/instances_status`
```

We explain in the page "_database_api/index.md" the different values for `DB_API_URL`. Can I ask you to update all occurrences of `db-api.scalingo.com` in `_database_api` folder with this variable? It seems it has been forgotten.